CAAppKitBridge/Source/GSCAData.h

```diff
@@ -32,14 +32,14 @@
   @public
   BOOL            _wantsLayer;
   BOOL            _isRootLayer;
-  CARenderer *    _renderer; // NIL if _isOriginalReciever == NO
-  CALayer    *    _layer;
+  CARenderer     *_renderer; /* NIL if _isOriginalReciever == NO */
+  CALayer        *_layer;
 
-  /* from libs-gui/Headers/AppKit/NSOpenGlView.h */
+  /* from libs-gui/Headers/AppKit/NSOpenGLView.h */
 
-  NSOpenGLContext 	  *  _GLContext;
-	NSOpenGLPixelFormat	*  _pixelFormat; //<- TODO: Check Apple behavior for propagating this
-	BOOL		             	 _prepared;
+  NSOpenGLContext 	        *_GLContext;
+	NSOpenGLPixelFormat	    *_pixelFormat; /* <- TODO: Check Apple behavior for propagating this */
+	BOOL		            _prepared;
```

Comment:
I don't know why I didn't leave a comment before, but this indentation is wrong.  Presumably it looks good for you because of the tab character (`\t`) being configured to two spaces in your editor. We generally don't use tab for indentation, precisely because systems don't agree on how to present it. For instance, GitHub seems to use eight spaces.